spec/relaton_ccsds/bibliography_spec.rb

```diff
@@ -1,17 +1,36 @@
-describe RelatonCcsds::Bibliography do
+class RelatonCcsds::TestHitCollection < RelatonCcsds::HitCollection
+  # override default index method to avoid index downloading
+  def index
```

Comment:
Because the new index file doesn't exist yet let's create the integration test after merging this branch. The GHA uses the GitHub version of this gem and will create an index file. We can implement the test then.